src/components/clusters/ClusterDetailsMultiRegion/components/Overview/InstallationLogView/InstallationLogView.jsx

```diff
@@ -264,7 +264,9 @@ LogWindow.propTypes = {
   cluster: PropTypes.shape({
     id: PropTypes.string,
     state: PropTypes.string,
-    subscription: { rh_region_id: PropTypes.string },
+    subscription: PropTypes.shape({
+      rh_region_id: PropTypes.string,
+    }),
```

Comment:
This error was causing an error to be displayed in the console when tests were run.